cpmonitor/tests/cleanupimages_test.py

```diff
@@ -0,0 +1,128 @@
+import os
+import shutil
+import tempfile
+from io import StringIO
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from cpmonitor.models import City
+
+
+class CleanupImagesTest(TestCase):
+    base_path = None
+    images_path = None
+    uploads_path = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.base_path = tempfile.mkdtemp()
+        cls.images_path = os.path.join(cls.base_path, "images")
+        cls.uploads_path = os.path.join(cls.images_path, "uploads")
+
+        cls.env_patcher = patch.dict(
+            os.environ,
+            {
+                "BASE_PATH": cls.base_path,
+                "IMAGES_PATH": cls.images_path,
+                "UPLOADS_PATH": cls.uploads_path,
+            },
+        )
+        cls.env_patcher.start()
+
+        super().setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+
+        shutil.rmtree(cls.base_path, True)
+
+        cls.env_patcher.stop()
+
+    def setUp(self):
+        super().setUp()
+
+        os.mkdir(self.images_path)
+        os.mkdir(self.uploads_path)
+
+        self.assertTrue(os.path.isdir(self.base_path))
+        self.assertTrue(os.path.isdir(self.uploads_path))
+        self.assertTrue(len(os.listdir(self.uploads_path)) == 0)
+
+        self.assertEqual(os.environ["BASE_PATH"], self.base_path)
+        self.assertEqual(os.environ["IMAGES_PATH"], self.images_path)
+        self.assertEqual(os.environ["UPLOADS_PATH"], self.uploads_path)
+
+    def tearDown(self):
+        super().tearDown()
+
+        shutil.rmtree(self.images_path, True)
+
+        self.assertFalse(os.path.isdir(self.uploads_path))
+        self.assertTrue(os.path.isdir(self.base_path))
+
+    def test_when_dry_run_then_do_not_delete_orphan_images(self):
```

Comment:
I like how you structured the tests and check a different business case with each test. :+1: